src/templates/prototypes/forms.pug

```diff
@@ -21,7 +19,7 @@ block content
 
           form(action='postcode')
             .form-group
-              +input('Search by postcode, for example; SW1A 0AA', 'postcode_main', 8, '[0-9a-zA-Z ]{5,}')
+              +input('Search by postcode, for example; SW1A 0AA', 'postcode_main', 'text', 8, '[0-9a-zA-Z ]{5,}', '- Special characters (!@£$%) are not permitted')
```

Comment:
Confirm this message with content - seems nonsensical to include that message when you can only enter numbers and letters anyway